test/commons/dom/find-nearby-elms.js

```diff
@@ -0,0 +1,41 @@
+describe('findNearbyElms', function () {
+  'use strict';
+  var fixtureSetup = axe.testUtils.fixtureSetup;
+  var findNearbyElms = axe.commons.dom.findNearbyElms;
+  var fixture;
+
+  function getIds(vNodeList) {
+    var ids = [];
+    vNodeList.forEach(function (vNode) {
+      if (vNode.props.id && vNode.props.id !== 'fixture') {
+        ids.push(vNode.props.id);
+      }
+    });
+    return ids;
+  }
+
+  beforeEach(function () {
+    fixture = fixtureSetup(
+      '<div id="n0" style="height:30px; margin-bottom:30px;">0</div>' +
```

Comment:
This test fails when debugging in a browser due to a Mocha element. 

<img width="946" alt="Test result for 'returns node from the same grid cell' showing failure due to 'mocha-stats' node being included in the results" src="https://user-images.githubusercontent.com/2433219/188750771-1b7ee543-3eb2-4fc3-8a1c-765b9b1a74ac.png">

Should probably set the width of these nodes to something small (like 30px) so it doesn't try to include extraneous elements outside of fixture.

Reply:
How do I get to this? And should we care?